config/management/operational/src/validator_set.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use libra_management::{error::Error, json_rpc::JsonRpcClientWrapper};
+use libra_types::{account_address::AccountAddress, validator_info::ValidatorInfo};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub struct ValidatorSet {
+    #[structopt(long, help = "JSON-RPC Endpoint (e.g. http://localhost:8080")]
+    host: String,
+    #[structopt(long, help = "AccountAddress to retrieve the validator set info")]
+    account_address: Option<AccountAddress>,
+}
+
```

Comment:
We'll have to come up with a better organizational structure than the tiny files (my fault) at some point.  Maybe by some sort of grouping.